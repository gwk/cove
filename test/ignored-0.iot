{
  'cmd': '$PROJ/cove.py test/ignored.py 0',
}
