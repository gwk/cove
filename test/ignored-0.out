----------------
Coverage Report:

__main__: ignored-{}.py:
   1
   2   from sys import argv
   3
   4
   5   def top(on):
   6     if on:
   7 !     inline(on)
   8 !     multi(on)
   9 !     partial(on)
  10
  11 | def inline(on): return on if on else -1 #!cov-ignore.
  12
  13   def multi(on):
  14 |   if on: #!cov-ignore.
  15 |     return on
  16 |   return -1 #!cov-ignore.
  17
  18   def partial(on):
  19 |   if on: #!cov-ignore.
  20 |     return on
  21 !   return -1 # never covered; this test assures that previous annotation is terminated.
  22
  23
  24   for a in argv[1]: top(int(a))

__main__: ignored-{}.py: 24 lines; 8 trivial; 16 traceable; 6 covered; 6 ignored; 0 ignored but covered; 4 not covered.
